src/frontend/src/eth/components/send/SendInfo.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import { getContext } from 'svelte';
+	import { SEND_CONTEXT_KEY, type SendContext } from '$icp-eth/stores/send.store';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+	import type { Erc20Token } from '$eth/types/erc20';
+	import Info from '$lib/components/ui/Info.svelte';
+
+	const { sendToken, sendPurpose } = getContext<SendContext>(SEND_CONTEXT_KEY);
+
+	let displayInfo: boolean;
+	$: displayInfo = ['convert-eth-to-cketh', 'convert-erc20-to-ckerc20'].includes(sendPurpose);
+
+	let sendTokenAsErc20: Erc20Token | undefined;
+	$: sendTokenAsErc20 = $sendToken.standard === 'erc20' ? ($sendToken as Erc20Token) : undefined;
+</script>
+
+{#if displayInfo}
+	<Info>
+		<p>
+			{#if sendPurpose === 'convert-eth-to-cketh'}
+				{$i18n.convert.text.cketh_conversions_may_take}
+			{:else}
+				{replacePlaceholders($i18n.convert.text.ckerc20_conversions_may_take, {
+					$ckErc20: sendTokenAsErc20?.twinTokenSymbol ?? 'ckETH'
```

Comment:
- Instead of ckETH should'nt we use something like `nativeEthereumToken.symbol` or something?

- Or maybe "ckETH" is fundamentally an incorrect fallback and we need something else?

Reply:
Theoretically, that fallback is just a proxy to make the type-interpreter ok: we would never arrive at that else condition if `twinTokenSymbol` is `undefined`, because we arrive at that part if we are in an ERC20 token with a ckERC20 counterparty.

The extreme correct way to do this, is to assert first if `twinTokenSymbol` is `undefined` and raise an error saying something like "inconsistent data, you should not have this screen, etc etc"

Reply:
I see your point but, even if edge case, potentially displaying "ckETH" is incorrect no? Should we just fallback to `''` or have two sentences?

Reply:
i prefer to fall back to `''`

however, can I do it in another PR, after? There is another place where it should be changed too

Reply:
I prefer that we fallback this one already to `''` as it feels incorrect and it's a new piece of code if you don't mind?